doc/additional_linux_details.rst

```diff
@@ -55,6 +55,13 @@ build directory by running::
    scan-build cmake --build build
 
 
+Debug compilation issues
```

Comment:
This is not specific to Linux.

Reply:
Moved to the end of the Tutorial page.
Done.